modules/nav.js

```diff
@@ -0,0 +1,21 @@
+export function showContent(sectionId) {
+  const sections = document.getElementsByClassName('content');
+
+  for (let i = 0; i < sections.length; i += 1) {
+    sections[i].style.display = 'none';
+  }
+
+  const selectedSection = document.getElementById(sectionId);
+  selectedSection.style.display = 'block';
+}
+
+export function setupNavigation() {
+  const links = document.querySelectorAll('.navbar a');
+  links.forEach((link) => {
+    link.addEventListener('click', (event) => {
+      event.preventDefault();
+      const sectionId = link.getAttribute('href').substring(1);
+      showContent(sectionId);
+    });
+  });
```

Comment:
- [ ] Ouch! this is close but as part of the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md) you have to replace each function definition with an ES6 arrow function so please make sure to implement that for the above `showContent` & `setupNavigation` functions. ✔

Reply:
Thank you for your time and valuable feedback. I appreciate your input and will consider it